### ISSUE TITLE
Koshy Individual Final Review Ticket

### ISSUE THREAD

**Issue by jli615** [jli615/lebroncs#47]

**Comment by jli615** [comment]
**TOTAL SCORE: 5.5/6**

Topic 1
Relation to CB topic: 1/1
Command of Technical Topic: 0.75/1
Vibe: 0.75/1
Total Score: 2.5/3

Topic 2
Relation to CB topic: 1/1
Command of Technical Topic: 1/1
Vibe: 1/1
Total Score: 3/3

Comments:
- good job going through line by line in the code to explain ascii lab
- good explanation of polymorphism and relationship between parent and child
- great diagram that helped viewer understand logic behind the ascii lab

Improvement
- explanation of the first topic on polymorphism could have been more detailed and organized

**Comment by ghost** [comment]
**TOTAL SCORE: 5.5/6**

Topic 1
Relation to CB topic: 1/1
Command of Technical Topic: 1/1
Vibe: 0.75/1
Total Score: 2.75/3

Topic 2
Relation to CB topic: 0.75/1
Command of Technical Topic: 1/1
Vibe: 1/1
Total Score: 2.75/3

Comments:
- Very good job explaining the technical aspect of each section code, it made it very easy to understand 
- The diagram you made for explaining the ascii blocking process is super helpful for getting a grasp on each part 
- I especially liked how you went through all parts of the code relating to polymorphism in order to define the behaviors of each singular section that brings about the outcome

Improvement:
- Could have used a better explanation on relationship to college board or purpose for the second topic(ascii)

**Comment by ryanmgds** [comment]
TOTAL SCORE: 5.5/6

Topic 1
Relation to CB topic: 1/1
Command of Technical Topic: 1/1
Vibe: 0.75/1
Total Score: 2.75/3

Topic 2
Relation to CB topic: 1/1
Command of Technical Topic: 1/1
Vibe: 0.75/1
Total Score: 3/3

Comments:

-great job going in depth with explaining code, really showed understanding
-I liked the diagram you created for the ascii block, helped me understand what you were doing
-well done in relating work to college board topics, work directly showed evidence of all college board topics related to

Improvement:

you could have gone more into depth on the first topic of polymorphism, at times was a bit hard to follow

**Comment by jm1021** [comment]
![image](https://user-images.githubusercontent.com/54686997/139540958-fba2b73e-fd0f-45ec-ab4b-e3fe478cde02.png)

Use of Static for Iter, very cool.  Diagram was the Awesome!  5.75

**Comment by jm1021** [comment]
1.5 for comments.
